components/calendar/src/any_calendar.rs

```diff
@@ -758,7 +758,7 @@ impl AnyCalendar {
         locale: include,
         options: skip,
         error: DataError,
-        #[cfg(skip)]
+        #[cfg(not(all()))]
```

Comment:
Suggestion: these `cfg(skip)`s in the invocations of `gen_any_buffer_data_constructors!` could be better modeled by changing the macro itself to not emit these functions.

Reply:
Yes, that's what I said in the comments, but I didn't want to figure that out whilst fixing nightly CI.